plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/extensions/IrBodyFiller.kt

```diff
@@ -218,6 +220,21 @@ private class DataFrameFileLowering(val context: IrPluginContext) : FileLowering
         return declaration
     }
 
+    private fun IrDeclarationWithName.nestedName() = StringBuilder().let { computeNestedName(this, it); it.toString() }
+
+    private fun computeNestedName(declaration: IrDeclarationWithName, result: StringBuilder): Boolean {
```

Comment:
How does this work when parents have backticks in the name? I suspect `` `Parent name`_`child name` `` won't compile, right?

Reply:
Not sure yet. There are a few places where backtick identifiers can break things. I plan to review it later

Reply:
alright, don't forget it ;) haha